cg/meta/backup/backup.py

```diff
@@ -294,31 +292,29 @@ def get_archived_flow_cell_path(self, query: list) -> Path:
             and FileExtensions.GPG in row
         ][ListIndexes.FIRST.value]
 
-        for dir in self.encrypt_dir.values():
-            re_archived_flow_cell_path: re.Pattern = re.compile(dir + ".+?(?=\s)")
-            arhived_flow_cell: Optional[re.Match] = re.search(
-                re_archived_flow_cell_path, flow_cell_query
-            )
-            if arhived_flow_cell:
-                archived_flow_cell_path: Path = Path(arhived_flow_cell.group())
-                LOG.info(f"Flow cell found: {archived_flow_cell_path}")
-                return archived_flow_cell_path
+        re_archived_flow_cell_path: re.Pattern = re.compile(self.encrypt_dir + ".+?(?=\s)")
```

Comment:
```suggestion
        re_archived_flow_cell_path: re.Pattern = re.compile(self.encrypt_dir + ".+?(?=\s)")
```
should we add some info what this pattern matches?

Reply:
Removed regexp and simplified using split instead -  fixed